app/build.gradle

```diff
@@ -129,6 +129,8 @@ dependencies {
     implementation 'se.emilsjolander:stickylistheaders:2.7.0'
     implementation 'com.github.chrisbanes:PhotoView:2.1.3'
     implementation 'me.dm7.barcodescanner:zxing:1.9.8'
+    implementation 'com.google.zxing:core:3.3.0'
+    implementation 'com.journeyapps:zxing-android-embedded:3.2.0@aar'
```

Comment:
any particular reason to not use the latest version?
I think we can use [the latest version](https://github.com/journeyapps/zxing-android-embedded/releases): 3.6.0, which should transitively pull in `zxing:core`, so we don't need to include it here